sdk/maps/azure-maps-geolocation/README.md

```diff
@@ -0,0 +1,202 @@
+# Azure Maps Geolocation Package client library for Python
+
+This package contains a Python SDK for Azure Maps Services for Geolocation.
+Read more about Azure Maps Services [here](https://docs.microsoft.com/azure/azure-maps/)
+
+[Source code](https://github.com/Azure/azure-sdk-for-python/tree/main/sdk/maps/azure-maps-geolocation) | [API reference documentation](https://docs.microsoft.com/rest/api/maps/geolocation) | [Product documentation](https://docs.microsoft.com/azure/azure-maps/)
+
+## _Disclaimer_
+
+_Azure SDK Python packages support for Python 2.7 has ended 01 January 2022. For more information and questions, please refer to <https://github.com/Azure/azure-sdk-for-python/issues/20691>_
+
+## Getting started
+
+### Prerequisites
+
+- Python 3.6 or later is required to use this package.
```

Comment:
I don't think we support Python 3.6 anymore?  This should be Python 3.7 or later, right @annatisch?